### ISSUE TITLE
Use of mj-hero results in broken email

### ISSUE THREAD

**Issue by EdwardSalter** [SebastianStehle/mjml-net#161]
After spending many hours attempting to convert a mjml-js solution over to MSML.NET, I have encountered an issue with the `<mj-hero >` component.

I have attempted to send an email that was generated from the following template:
```xml
<mjml>
  <mj-body>
    <mj-hero>
      <mj-text>
        Hello world
      </mj-text>
    </mj-hero>
  </mj-body>
</mjml>
```

This generates the following (shortened) HTML:
```html
  
  <!--[if mso | IE]>
  <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" width="600" style="width:600px;">
    <tr>
      <td style="line-height:0;font-size:0;mso-line-height-rule:exactly;">
        <v:image xmlns:v="urn:schemas-microsoft-com:vml" style="border:0;mso-position-horizontal:center;position:absolute;top:0;width:600px;z-index:-3;">
          <![endif]-->
          <div style="margin:0 auto;max-width:600px;">
            <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
              <tbody>
                <tr style="vertical-align:top;">
                  <td height="0" style="background:#ffffff;background-position:center center;background-repeat:no-repeat;padding:0px;padding-bottom:0px;padding-left:0px;padding-right:0px;padding-top:0px;vertical-align:top;">
                    <!--[if mso | IE]>
                    <table border="0" cellpadding="0" cellspacing="0" width="600" style="width:600px;">
                      <tr>
                        <td>
                          <![endif]-->
                          <div class="mj-hero-content"  style="margin:0px auto;">
                            <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;margin:0px;">
                              <tbody>
                                <tr>
                                  <td>
                                    <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;margin:0px;">
                                      <tbody>
                                        <tr>
                                          <td align="left" style="font-size:0px;padding:10px 25px;padding-bottom:10px;padding-left:25px;padding-right:25px;padding-top:10px;word-break:break-word;">
                                            <div style="font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:1;text-align:left;color:#000000;">
Hello world
                                            </div>
                                          </td>
                                        </tr>
                                      </tbody>
                                    </table>
                                  </td>
                                </tr>
                              </tbody>
                            </table>
                          </div>
                          <!--[if mso | IE]>
                        </td>
                      </tr>
                    </table>
                    <![endif]-->
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
          <!--[if mso | IE]>
        </td>
      </tr>
    </table>
    <![endif]-->
```

When sent and opened in Outlook, this does not render at all.
After using the same MJML and running it through the Node version of the tool, I have managed to track down the difference in the generated output which basically boils down to this element:
MSML.NET Version:
`<v:image xmlns:v="urn:schemas-microsoft-com:vml" style="border:0;mso-position-horizontal:center;position:absolute;top:0;width:600px;z-index:-3;">`

MSML.JS Version:
`<v:image style="border:0;mso-position-horizontal:center;position:absolute;top:0;width:600px;z-index:-3;" xmlns:v="urn:schemas-microsoft-com:vml" />`

Note the self closing tag in the JS version.
After fiddling about with some EML files, I have confirmed that the missing closing tag is what causes the entire email to not render correctly. Add in the slash and everything is fine.

I did some digging around the code and I think it is a simple fix, simply modifying the following line:
https://github.com/SebastianStehle/mjml-net/blob/ad4c1d78f1cc8a61187a48dde77a3844d0eb6762/Mjml.Net/Components/Body/HeroComponent.cs#L131
to instead be 
`                renderer.StartElement("v:image", true) // Style: outlook-image`
to automatically self-close the element.

I debugged the current Hero Component tests and confirmed that before this change, the result has no closing tag and with the change, it does.
Interestingly, the HTML Asserts do not seem to check whether or not the element closes itself so despite the expected HTML file showing the element correctly closed and the actual result not being closed, the test still passes.

**Comment by SebastianStehle** [comment]
Thanks a lot for your find. 

The HTML assets checks on semantic issues only, so I guess it is just the same for it. Would your provide a PR?

**Comment by SebastianStehle** [comment]
See: https://github.com/AngleSharp/AngleSharp.Diffing/discussions/30

**Comment by EdwardSalter** [comment]
@SebastianStehle 
I have created the PR as requested. It only contains the change outlined above and no test changes to the issue you have mentioned in AngleSharp.Diffing.
I'm not sure there is really any other way of testing this other than resorting back to string comparisons which I am sure will break for a myriad of reasons.
I have debugged the hero tests again both before and after the change to double check the v:image tag is correctly closed.
